Plugins/GitUIPluginInterfaces/EffectiveGitSetting.cs

```diff
@@ -0,0 +1,15 @@
+namespace GitUIPluginInterfaces
+{
+    public record struct EffectiveGitSetting(GitConfigStatus? Status, string Value)
+    {
+        public static implicit operator (GitConfigStatus? status, string value)(EffectiveGitSetting value)
```

Comment:
Avoid _value_ for arguments, there are several layers with the same name

Reply:
What do you suggest instead of the "Value" from the config entry and still having status and config key?

Reply:
or something, instead of the contextual keyword https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/value
```suggestion
        public static implicit operator (GitConfigStatus? status, string errorMessage)(EffectiveGitSetting gitSetting)
```

Reply:
In this instance `Value` represents git setting value, so I think the name is appropriate. I would ask for xml-docs clearly explaining the nature and the purpose for all public API though.

Reply:
Yep. Def was going to do so once feedback was provided.

Reply:
But Value vs value is confusing here, i specific name would add value, this is not a generic interface. What is it that the value holds?

Reply:
https://github.com/gitextensions/gitextensions/pull/11146#discussion_r1292905870